src/SprykerEco/Zed/Payone/Business/Api/TransactionStatus/TransactionStatusRequest.php

```diff
@@ -978,15 +978,15 @@ public function getClearingInstructionnote()
      *
      * @return void
      */
-    public function setClearingLegalnote($clearing_legalnote)
+    public function setClearingLegalnote(string $clearing_legalnote): void
```

Comment:
Please change all arguments in this class to be camel cased.

Reply:
Fixed for this and other cases.